### DISCUSSION TITLE
Distribution Groups and Contacts

### DISCUSSION THREAD

**Discussion by ShterMcGavin** [netbox-community/netbox#17647] [Ideas]
Would like to be able to create Distribution Groups in AD then use those Distribution Groups to populate contact assignments within Netbox.

Example. Users A, B, C, and D are within an AD Distribution Group called siteX. Automation should then be able to pull the contacts within the distribution group for siteX to populate the contacts for the site siteX on Netbox. When the distribution group siteX is updated, so say we remove user D, the contacts within the site siteX on NetBox should reflect the changes within the Distribution Group. This would allow just a simple update within Distribution groups to change the contact list for an associated object.

I have been unable to locate an integration that does what this is requesting. So if there is already one, could someone please point me in the right direction?

**Comment by sol1-matt** [comment]
good workaround for this here https://github.com/netbox-community/netbox/discussions/15064#discussioncomment-8388734
